### PR TITLE
Update at3plusHeaderMap

### LINKED ISSUE #7420: Gundam AGE Universe Accel missing voice since ffmpeg update
https://github.com/hrydgard/ppsspp/commit/4ebdf49fb68c72473efeb4279042d082975755f8

```
CRI Server M I[ME]: HLE\sceAtrac.cpp:788 1=sceAtracGetAtracID(4096)
CRI Server M I[ME]: HLE\sceAtrac.cpp:2077 sceAtracLowLevelInitDecoder(1, 09fdc8b0)
CRI Server M I[ME]: HLE\sceAtrac.cpp:2086 Channels: 1 outputChannels: 1 bytesperFrame: 118
CRI Server M W[ME]: HLE\sceAtrac.cpp:2115 This is an atrac3+ mono audio (low level)
CRI Server M E[ME]: HLE\sceAtrac.cpp:2123 AT3plus header map lacks entry for bpf: 280  channels: 1
CRI Atom Atr I[ME]: HW\MediaEngine.cpp:85 FF: Frame data doesn't match channel configuration!
CRI Atom Atr E[ME]: HLE\sceAtrac.cpp:415 avcodec_decode_audio4: Error decoding audio -1094995529 / bebbb1b7
```

previous log
```
CRI Server M I[ME]: HLE\sceAtrac.cpp:750 1=sceAtracGetAtracID(4096)
CRI Server M I[ME]: HLE\sceAtrac.cpp:2034 sceAtracLowLevelInitDecoder(1, 09fdc8b0)
CRI Server M I[ME]: HLE\sceAtrac.cpp:2043 Channels: 1 outputChannels: 1 bytesperFrame: 118
CRI Server M W[ME]: HLE\sceAtrac.cpp:2072 This is an atrac3+ mono audio (low level)
CRI Server M E[ME]: HLE\sceAtrac.cpp:2080 AT3plus header map lacks entry for bpf: 280  channels: 1
CRI Server M I[ME]: HW\MediaEngine.cpp:85 FF: Channel layout 'mono' with 1 channels does not match specified number of channels 2: ignoring
specified channel layout
```

### PR DESCRIPTION
**Pull request by daniel229**
Fixes https://github.com/hrydgard/ppsspp/issues/7420 ,still thinking<code> { 0x0180, 0x1, 0x0 },</code>  actually is <code>{ 0x0118, 0x1, 0x0 }</code>

### CODE REVIEW

File: Core/HLE/sceAtrac.cpp [hunk 1]
```diff
@@ -1964,6 +1964,8 @@ static const u16 at3plusHeaderMap[][3] = {
 	{ 0x4524, 0x0, 0x0 },
 	{ 0x02E8, 0x1, 0x0 },
 	{ 0x5C24, 0x0, 0x0 },
+	{ 0x0118, 0x1, 0x0 },
+	{ 0x2224, 0x0, 0x0 },
```
Comment (unknownbrackets) [inline Core/HLE/sceAtrac.cpp]: This is already above (0x2224)?

-[Unknown]

Comment (daniel229) [inline Core/HLE/sceAtrac.cpp]: Yeah,so I think the above one is 0x0118,not 0x0180.0x2224 should make pair with 0x0118.
you can see those pair
```
{ 0x0178, 0x1, 0x0 }, { 0x2E24, 0x0, 0x0 }, 
{ 0x0178, 0x2, 0x0 }, { 0x2E28, 0x0, 0x0 },
```
```
 { 0x0230, 0x1, 0x0 },{ 0x4524, 0x0, 0x0 },
 { 0x0230, 0x2, 0x0 },{ 0x4528, 0x0, 0x0 },
```
```
{ 0x02E8, 0x1, 0x0 },{ 0x5C24, 0x0, 0x0 },
{ 0x02E8, 0x2, 0x0 },{ 0x5C28, 0x0, 0x0 },
```

Comment (hrydgard) [inline Core/HLE/sceAtrac.cpp]: Well, no need to make a duplicate of the 0x2224 one for that. Just add 0x0118 next to 0x0180, good enough, the pairs don't matter.

Comment (daniel229) [inline Core/HLE/sceAtrac.cpp]: the https://github.com/hrydgard/ppsspp/pull/7530/files#diff-a4d8150342953ac0053b961c4601e20fL2013 
it's i+=2
If don't add a pair, the following thing would miss.

Comment (daniel229) [inline Core/HLE/sceAtrac.cpp]: This one
for (int i = 0; i < at3plusHeaderMapSize; i += 2) {

Comment (unknownbrackets) [inline Core/HLE/sceAtrac.cpp]: That's... confusing.

Shouldn't it just be `{ 0x00C0, 0x1, 0x1724 },`, `{ 0x0180, 0x1, 0x2224}` etc. then?  But I guess that's not the problem of this pull request.

-[Unknown]

**Comment by unknownbrackets** [comment]
This should've affected ダンボール戦機Ｗ (Danball Senki W) as well as 机动战士高达AGE Cosmic Drive and 機動戦士ガンダムAGE ユニバースアクセル.

-[Unknown]

**Comment by daniel229** [comment]
Yeah,also fixes missing voices in Danball Senki W I tested.

**Pull request merged**